apps/test/unit/templates/levelSummary/SummaryResponsesTest.jsx

```diff
@@ -125,7 +125,7 @@ describe('SummaryResponses', () => {
       }
     );
 
-    expect(wrapper.find('ToggleSwitch').length).toBe(1);
+    expect(wrapper.find('Toggle').length).toBe(1);
```

Comment:
This is only a temporary change.  I am working on updating this whole test to use React Testing Library (in another PR)